packages/next/package.json

```diff
@@ -64,6 +64,7 @@
     "hoist-non-react-statics": "3.2.0",
     "htmlescape": "1.1.1",
     "http-status": "1.0.1",
+    "isomorphic-fetch": "2.2.1",
```

Comment:
@timneutkens is this ok? it should't be `isomorphic-unfetch`? Or even not needed?